mne/decoding/time_gen.py

```
@@ -203,16 +224,31 @@ def predict(self, epochs):
         if not hasattr(self, 'estimators_'):
             raise RuntimeError('Please fit models before trying to predict')
 
-        # clean attributes
+        # Check predict mode
+        if self.predict_mode not in ['cross-validation', 'mean-prediction']:
+            raise ValueError('`predict_mode` must be a str, "mean-prediction" '
+                             'or "cross-validation"')
+
+        # Check that training cv and predicting cv match
+        if self.predict_mode == 'cross-validation':
+            n_est_cv = [len(estimator) for estimator in self.estimators_]
```

why would a user do that? Is there a legitimate use case? Otherwise, we can throw an error when the user tries to change the `estimators_` attribute